### PR TITLE
netlify deployment guide

### LINKED ISSUE #202: Guide on deploying with Netlify
## Type of Change
- [ ] New Feature Request
- [x] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
It would be great to have a documentation section on how to deploy with Netlify, start to finish, using Greenwood.

## Details
Topic to include
1. What you need (GitHub repo, etc)
1. Netlify Configuration
1. DNS Config
1. Greenwood Config
1. GitHub (deploy previews and continuous deploy from master)

### PR DESCRIPTION
**Pull request by aholtzman**
## Related Issue

resolves #202 

## Summary of Changes

added guide for simple Netlify deployment

### CODE REVIEW

File: www/pages/guides/netlify-deploy.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: 'Deploy on Netlify'
+menu: side
+linkheadings: 3
+index: 1
+---
+
+## Deploying your site on Netlify
+
+Requires an account with [Netlify](https://www.netlify.com) liked to your GitHub account. If you need to sign up, the process is simple as is the deployment.
```
Comment (thescientist13) [inline www/pages/guides/netlify-deploy.md:10]: >  liked to your GitHub account

_linked_ to your GitHub account

File: www/pages/guides/netlify-deploy.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+After you have created your Greenwood project and pushed your code to a GitHub repo...
+
+- log in to your Netlify account and proceed to the dashboard.
```
Comment (thescientist13) [inline www/pages/guides/netlify-deploy.md:14]: Just a nitpick, but maybe we could have consistent capitalization for the start of each list item?

**Comment by aholtzman** [comment]
I do find images/screen shots helpful with these sorts of guides when the pages where they are referencing are pretty confusing (looking at you AWS) but the UI for Netlify (also Now) is really clean and intuitive, so I do not feel like they are necessary. Easy enough to add them, but it may set a president where all of the guides will need them - worth thinking it over weighing the need/convenience/bloat with adding a bunch of images.

File: www/pages/guides/netlify-deploy.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+index: 1
+---
+
+## Deploying your site on Netlify
+
+Requires an account with [Netlify](https://www.netlify.com) linked to your GitHub account. If you need to sign up, the process is simple as is the deployment.
+
+After you have created your Greenwood project and pushed your code to a GitHub repo...
+
+- Log in to your Netlify account and proceed to the dashboard.
```
Comment (thescientist13) [inline www/pages/guides/netlify-deploy.md:14]: Sorry, same as with the other one, should have caught this before, re: using an ordered / numbered list would be more appropriate?

File: www/pages/guides/netlify-deploy.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: 'Deploy on Netlify'
+menu: side
+linkheadings: 3
+index: 1
```
Comment (thescientist13) [inline www/pages/guides/netlify-deploy.md:5]: AWS guide already has an index of 1

Comment (thescientist13) [reply www/pages/guides/netlify-deploy.md:5]: @aholtzman 
This still seems to apply I think, so this and AWS guide need to figure out a new ordering

Comment (aholtzman) [reply www/pages/guides/netlify-deploy.md:5]: Yeah - the AWS order was set before Netlify doc was created, but the Netlify should be higher based on importance to users.

**Comment by aholtzman** [comment]
sorry about that, pre coffee change. was some wonkiness after I did a pull.

**Pull request merged**